src/components/LoadDatasetButton.tsx

```diff
@@ -219,11 +224,11 @@ export default function LoadDatasetButton(props: LoadDatasetButtonProps): ReactE
   });
 
   // Get the URLs (keys) of any recent datasets that match the currently selected urlInput.
-  const matchingKeys = recentDatasets.filter(({ label }) => label === urlInput).map(({ url }) => url);
+  const matchingKeys = recentCollections.filter(({ label }) => label === urlInput).map(({ url }) => url);
   const datasetsDropdownProps: MenuProps = {
     onClick: (info) => {
       // Set the URL input to the label of the selected dataset
-      const dataset = recentDatasets.find(({ url }) => url === info.key);
+      const dataset = recentCollections.find(({ url }) => url === info.key);
```

Comment:
just for my own knowledge: what's the definition of a collection and a dataset? It seems like there are both things, but here it seems like you're using the terms equivalently?

Reply:
Oops, I totally forgot to update the `dataset` variable here, I'll make that change. The change here was made because `recentDatasets` is _usually_ storing collections instead of datasets (though maybe this should be `recentResources`?).

- A dataset is a single image sequence + feature/data JSON files, all indexed by a manifest JSON file. 
- A collection is a JSON file containing paths to one or more datasets-- e.g. a collection of datasets. When a collection is loaded, you can switch between its datasets in the dropdown.

The colorizer tool can load in **both** collections and datasets directly, though internally it treats both as collection objects. (If you load in a dataset, it wraps it in a dummy collection and treats it like a collection with only a single dataset inside.)

Reply:
Some of the language here is legacy from when we used collections/datasets interchangeably. I usually prefer using the word "dataset" for external users since it's a bit more intuitive.